### ISSUE TITLE
Medium uses wrong AAT concept

### ISSUE THREAD

**Issue by VladimirAlexiev** [american-art/aac_mappings#37]
You map Medium to [aat:300264237](http://www.getty.edu/vow/AATHierarchy?find=material+components&logic=AND&note=&english=N&prev_page=1&subjectid=300264237) "material components" which has such curious children as atoms, electrons, eggshell, ossein (bone material) etc. How does "oil on canvas" fit in here?

Use `aat:300163343` "medium (artists' materials)".

Furthermore, representing medium/material as Linguistic Object makes no sense to me at all.

**Comment by workergnome** [comment]
In a perfect world, we'd have broken out medium for each element of the object, and we do have a mapping for people who have that level of specificity in their data.  But what we're finding is a bunch of people who don't have medium elements, but instead have a written description of the medium: a "medium statement", if you will.  We're using "linguistic object" to define that statement, and then we have to differentiate between various types of statements that are made, including dimension statements and credit lines.   So we're using AAT terms to classify them.

**Comment by azaroth42** [comment]
Per the scope notes, canvas doesn't fall under 300163343, which is just the "materials applied to the support" -- the ink, oil or watercolor, not the paper or canvas.

As David says, there's a lot of different stuff in there that we could parse out to component PhysicalObjects with Materials ... but not today. Or this year. Or for furniture, ever.

**Comment by VladimirAlexiev** [comment]
- You're right, 300163343 is only Medium while the data includes Medium+Support.
- Please use [aat:300014842](http://www.getty.edu/vow/AATHierarchy?find=medium&logic=AND&note=&page=1&subjectid=300014842) Artists materials, which has children Medium and Support.
- aat:300264237 is "Parts or components of materials". I don't know what are "parts of materials" (always thought materials have no countable parts), but don't the children worry you? Atoms? Electrons? 
- People do use all kind of weird materials to make artworks, eg see here a [painting made from barley](http://petrov.com.ua/GoldCalf/pages/32.htm). So if you dislike 300014842 Artists materials, please use [aat:300010358](http://www.getty.edu/vow/AATHierarchy?find=material+components&logic=AND&note=&page=1&subjectid=300010358) Materials.

**Comment by azaroth42** [comment]
I'm fine with 300010358.  @workergnome?

**Comment by workergnome** [comment]
I'm fine with that—the question is that since this is part of a change to things already modeled, we need to get ISI on board.  The other option is to change it in v 1.1, which is the imaginary version I'm thinking we'd publish as a recommendation.  My preference is to fix it—it's easy for me at this point, but I only have to change it two places.  ISI would have to change it in a bunch of places.

**Comment by azaroth42** [comment]
I think we fix in documentation and note that the AAC 1.0 model used 300264237 incorrectly. Then stack up the changes and prioritize which are essential to make, and which just nice to have.  This seems like a nice to have.

**Comment by workergnome** [comment]
Indeed.  I've been noting changes like this, and I may fix many of them in the CONSTRUCT query the browse app is running to build the JSON-LD representation.  My suggestion would be to keep issues like this open on this repository, and we'll evaluate them en-masse once @VladimirAlexiev has completed his review.

That work for you two?

**Comment by azaroth42** [comment]
@workergnome Can you create a label for it and assign to the issue?

**Comment by workergnome** [comment]
(not an admin, so I can't add new labels, but I created a milestone before transferring, so I'll add that.)

**Comment by VladimirAlexiev** [comment]
Eleanor made it very clear: v 1.1.
Whatever is discussed in this project will not be remapped by ISI in the individual museum projects.
Unless we find serious omissions in the mapping: then ISI should go back and add.
